sonar-db/src/test/java/org/sonar/db/version/AlterColumnsBuilderTest.java

```diff
@@ -115,6 +116,20 @@ public void fail_with_ISE_if_no_column() {
     new AlterColumnsBuilder(new H2(), TABLE_NAME).build();
   }
 
+  @Test
+  public void fail_with_IAE_if_default_value_is_defined() {
+    thrown.expect(IllegalArgumentException.class);
+    thrown.expectMessage("Default value is not supported on alter of column 'enabled'");
+
+    new AlterColumnsBuilder(new H2(), TABLE_NAME)
```

Comment:
written like this, the fact that the exception is thrown by `updateColumn` is not that obvious.
I see two ways to improve this (not exclusive):

1. rename method to start with `updateColumns_fails`
2. move the columnDef to a variable defined before the `thrown` statements